fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirUseAccessToken.java

```diff
@@ -0,0 +1,304 @@
+package io.vantiq.extsrc.fhirAssembly;
+
+import static io.vantiq.extsrc.fhirAssembly.TestFhirAssembly.SUB_USER;
+import static io.vantiq.extsrc.fhirAssembly.TestFhirAssembly.TEST_SERVER;
+import static io.vantiq.extsrc.fhirAssembly.TestFhirAssembly.performSetup;
+import static io.vantiq.extsrc.fhirAssembly.TestFhirAssembly.performTeardown;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqResponse;
+
+import org.apache.http.Header;
+import org.apache.http.HttpHost;
+
+import org.apache.http.entity.StringEntity;
+import org.apache.http.localserver.LocalServerTestBase;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.protocol.HttpRequestHandler;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
```

Comment:
Makes use of an apache test framework that sets up an http server for each test.  Using that, we can make our calls & return the headers supplied, which we can then validate.